src/sync/partial_sync.cpp

```diff
@@ -214,26 +200,61 @@ struct ResultSetsColumns {
     size_t matches_property;
 };
 
-bool validate_existing_subscription(Table& table, ResultSetsColumns const& columns, std::string const& name,
+// Validate the subscription about to be created against existing subscription.
+// If an existing subscription already exists that matches the one  we are about to create, the
+// index of that Subscription is returned. If no current matching subscription exists `npos` is
+// returned.
+static size_t validate_existing_subscription(Table& table, ResultSetsColumns const& columns, std::string const& name,
                                     std::string const& query, std::string const& matches_property)
```

Comment:
A bunch of spots in this function update the first line of a multi-line function call without adjusting the indent of the following lines to match.